unfoldingWord/translationCore
# New Stepper Implementation

### This pull request addresses:
- Refactor the stepper design to look like new layout mockup. 

### How to test this pull request:
- Open the app click the `Get Started` button and click next button to test stepper.
- stepper should look like in the mockups and color should change as you go to next or previous step.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1929)
